docker-compose.yml

```
@@ -23,7 +23,7 @@ services:
       # Our MGA ingest makes some *very* large transactions
       - "max_wal_size=2GB"
     ports:
-      - "5432:5432"
+      - "127.0.0.1:5432:5432"
```

I recommend adding a comment about this because I think it is the only occurrence of it in any of the NMDC repositories.

```suggestion
      # We specify `127.0.0.1` here so that other computers on the same LAN cannot
      # access the container, even when the host's firewall is configured to allow 
      # incoming connections to Docker.
      # Reference: https://github.com/compose-spec/compose-spec/blob/main/spec.md#short-syntax-3
      - "127.0.0.1:5432:5432"
```

---

Good suggestion, I just added in that comment